pom.xml

```diff
@@ -58,8 +61,8 @@
         <!-- build.config.version>9</build.config.version-->
         <bundle.plugin.version>2.5.3</bundle.plugin.version>
         <cdi.tck-1-2.version>1.2.10.Final</cdi.tck-1-2.version>
-        <cdi.tck-2-0.version>2.0.6.SP1</cdi.tck-2-0.version>
-        <cdi.tck-3-0.version>3.0.0-M1</cdi.tck-3-0.version>
+        <cdi.tck-2-0.version>2.0.7.SP1</cdi.tck-2-0.version>
```

Comment:
I think `cdi.tck-2-0.version` and `cdi.tck-1-2.version` properties can actually be removed (and along with them the profiles that consumed them) - we will no longer be able to execute older TCKs in Weld 4 due to package changes.